relecov_tools/wrapper.py

```diff
@@ -295,7 +296,21 @@ def upload_files_from_json(invalid_json, remote_dir):
         else:
             self.log.info("No invalid samples in %s", key)
             stderr.print(f"[green]No invalid samples were found for {key} !!!")
-        log_filepath = os.path.join(local_folder, str(key) + "_metadata_report.json")
+
+        logmod = BaseModule(output_dir=local_folder, called_module="logs_to_excel")
+
+        # Set batch ID
+        try:
+            batch_date = datetime.strptime(os.path.basename(local_folder), "%Y%m%d")
+        except Exception:
+            batch_date = logmod.basemod_date
+        logmod.set_batch_id(batch_date)
+
+        # Generate standardized filename
+        base_filename = f"logs_to_excel_{str(key)}"
+        log_filepath = os.path.join(
+            local_folder, logmod.tag_filename(base_filename) + "_metadata_report.json"
+        )
```

Comment:
Why do you name the file with "logs-to-excel" if its the wrapper module? Anyway I dont understand why do you change anything in this module. The issue refers to logs-to-excel

Reply:
This suggestion should be enough change I'd say. The resulting name would be:
 `COD-XXXX-TES-TING_batchid_hex_metadata_report.json/xlsx`

```suggestion
        log_filepath = os.path.join(
            local_folder, key + self.tag_filename("_metadata_report.json")
        )
```